### PR TITLE
Fixed missing ID in Trusted Shops product AggregateRating schema.org data.

### PR DESCRIPTION
**Pull request by Mauricio-Urrego**
### Ticket
- [External Trusted Shops JavaScript does not inject product ID into product aggregateRating (GACO loses review snippets, according to GSC)](https://app.asana.com/0/809933051638353/1175128107644777/f)

### Description
-

### CODE REVIEW

File: assets/scripts/main.js [hunk 1]
```diff
@@ -30,4 +30,38 @@
...
+        childList: true,
+        subtree: true,
+      });
+    });
+  }
+
+  waitForElm('#trustedshops-productreviews-sticker-wrapper').then(() => {
+    const TargetScripts = document.querySelectorAll('head > script[type="application/ld+json"]');
+    TargetScripts.forEach((TargetScript) => {
+      TargetScript.innerText = TargetScript.innerText.replace(',"@type":"Product","name"', `,"@id":"${location.protocol}//${location.host}${location.pathname}#product","@type":"Product","name"`);
```
[the remark this answers is not in the document]
Comment (Mauricio-Urrego) [reply assets/scripts/main.js]: We don't need the index position of the string if it finds the string so I have opted for an includes() check instead.

**Review by sun — CHANGES_REQUESTED** [review]
Looks good now. 👍  Please bump the minor version as part of this PR and also add the compiled assets.

File: package.json [hunk 2]
```diff
@@ -1,5 +1,5 @@
 {
-  "version": "1.7.1",
+  "version": "1.7.2",
```
Comment (sun) [inline package.json:2]: Please also here:
https://github.com/netzstrategen/woocommerce-reputations/blob/ba9e47f97e5cc0aa2c09cd2753bd639c50298ae3/plugin.php#L6

**Review by sun — APPROVED** [review]
I corrected PR title. Please also change the version of the plugin and squash-merge using the corrected PR title as commit message.

**Pull request merged**